packages/accessible-react-app/src/css/forms.css

```diff
@@ -22,9 +47,9 @@
 .ac-form__input,
 .ac-form__textarea {
   border: 1px solid #666666;
```

Comment:
realizing this color was already there, it would be nice to update to a variable just for consistency